### PR TITLE
Release b0147bdbed9f25212408e0468a475289e80e0406

### PR DESCRIPTION
**Pull request by Ark-kun**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/1238)

### CODE REVIEW

**Comment by Ark-kun** [comment]
>We should look at a better strategy for this though.

I'll implement the better way soon. The images/components won't be updated unless the code was changed.

>Either use constant URLs which get mapped by backend registry, or define global variables which can be referenced everywhere.
>Current approach is very error prone

Current approach provides a choice between exact reproducibility and using the latest version.

File: components/dataflow/predict/component.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ outputs:
   - {name: Predictions dir,     type: GCSPath,  description: 'GCS or local directory.'} #Will contain prediction_results-* and schema.json files; TODO: Split outputs and replace dir with single file # type: {GCSPath: {path_type: Directory}}
 implementation:
   container:
-    image: gcr.io/ml-pipeline/ml-pipeline-dataflow-tf-predict:e20fad3e161e88226c83437271adb063221459b9
+    image: gcr.io/ml-pipeline/ml-pipeline-dataflow-tf-predict:b0147bdbed9f25212408e0468a475289e80e0406
```
Comment (IronPan) [inline components/dataflow/predict/component.yaml]: why this sha instead of e8524eefb138725fc06600d1956da0f4dd477178?

Comment (Ark-kun) [reply components/dataflow/predict/component.yaml]: We want to release component images as from https://github.com/kubeflow/pipelines/commit/b0147bdbed9f25212408e0468a475289e80e0406 (as the title indicates).
https://github.com/kubeflow/pipelines/commit/e8524eefb138725fc06600d1956da0f4dd477178 is the commit that does this (updates images to b0147bdbed9f25212408e0468a475289e80e0406).
A commit cannot reference itself >_<

P.S. This PR has 3 commits.

**Pull request merged**